### DISCUSSION TITLE
Auto-Labeling Image data

### DISCUSSION THREAD

**Discussion by Saeed11b95** [doccano/doccano#2016] [General]
Hi There.
I am trying to setup a custom REST for auto labeling image classification data. However, i am unable to post image data to my REST endpoint.
Screenshots are attached for reference. Help would be appreciated.
Thanks in advance
![Screenshot from 2022-10-03 14-25-36](https://user-images.githubusercontent.com/91561697/193545547-e9df2301-2dcf-4d68-b794-475c4c3bc177.png)
![Screenshot from 2022-10-03 14-25-48](https://user-images.githubusercontent.com/91561697/193545554-311be393-aa47-476a-8c1d-dc6360ac9a5f.png)

**Comment by david-engelmann** [comment]
@Saeed11b95 What type of project did you create?
